public/wp-content/themes/glific/dist/images/blog-delaut-image.svg

```diff
@@ -1 +1,2 @@
c2QmEJoESICRIKNnS49ibUU6drCNKUrfc6Di8Yr/rdNT+7PmtvgokEud4MoV31ZRFgLZ7lq5ZT3MX/2VYr+M7IXpMRJYNm/+zQSqEwGyhzZ3+f/g9mcmwvxVSxS7eIa1X5JmFaVnvqImP1OaHvJbcZcMxd/vF7a55it/MjRJbvBuGnBNlboh81VHPscEzESAVjU/XbkY3lz0kSHzZpXaJklLLoOGI9LT01V58bamUKNWo2VclNzgEvDoXgmahwPUWyPb7Neh01ay8tqyYVMPsfgQE6ieBMiNHXl9IovF5FPA5dJuqVoNOdRH25BYr+4AsmirJj7Fsa5Qw8rTMLRoVQbIZ4vUtlc4XiPswZH/g9ZNWwL5aiQnG2T6mMw004e8FnFgAqFCoNRRqvinIB8VpDBLZtD3/r4fyHM6ORoW9cupiY8Eu6JqRPc+u/SXyg44fGRkaaFG7ZILS6FobWZQBJsPjnyKCTABIwmgQLOH2wf50kfzVpylVj89NYIUc6P64+R+on4dNk/58zEmwASCS4CGnNRDExFoVFPLCzVqhBRuh6i0VkBmwDkwASZgYQK4KEBzaFqHnGVbHBJCTWkQmgCP7NkCwpLrlW0f/2YCTMAqBFBtg1Y5tSwKeGpamKeDlj2GLmVo87u9bhyUbDkAcokBWsyWhcEVZwIWIYA7BUixtiB97xfpoM96B7U4dHpqZa6fvVECRA5qA7baPM9WBgv/ZAKmI4DmuH/ArU+dSaAZVbmQFGoEh7ZURfVPhohO+s0WGQWb82ECTMBNAM0H2WyTe0gNe6rZ+uROpeY7tIafFVuMw9GwlHpgb1ILSrYdBnKSzIEJMIGqJUAGHm0Rtkm0upkOew2vTGgLtT9xkdpH5OXNwZmdCI6dx8CZk284SM6QCTAB3wTIpwBurJ7qzWKt79Tqz1YLoebGoZgG7xenCDXHr8fBefyc+xR/MwEmECAC5MYO0OuTJycpgSiyWgk1N0DFS1WfJHCdKgDH/pPKsJRXSt10+JsJGEKgFHtm75Kj4Yp+OQ3J3Ucm1VKouXmQw+SI2k0AUA3EefQsOA6cAueJPGWzvDsOfzMBJiBAQIK30eDEBIGUupNUa6F2gR4q7tob1/rDaxVa/3Ci5Q9X9jkUcPngOl0AgKaOODABJqCBgARV5kWchVrF60QCrj665cNPeHs8iULOlVcE8rli5duVVwzy+WKAUjS14nCC7HChtRD8HSDTKxWrx/+ZQFAIoOktKcyG9zaaMBN4p6Px/Yyg1NNDISzUPEApdwiFnK1mDAB+9DnpK5cr/2ECpifgPHYOitdlCdVTkl1VJtRCVvlW6EpwIibABC4QcB0/e+G3lh+42nm0MD3rsJY0RsZloWYkTc6LCYQKAZxHdhwUVFaXYE1VYmChVpX0uWwmYFICpMMpF6tyCVC5BZK8uvLB4B1hoRY81lwSE7AMAce+k+J1lcNYqInT45RMgAkYTYB8fpDepmDINHqDutZ6cE9NKzGOzwRCnEDJ7uPCLZQkmCec2KCELNQMAsnZMIFQIOA6UwjOQ4ILBAQgzPZhVXNgoVbVV4DLZwImIlCy9RAq2wpo22IbsJf2fdHKvfurujks1Kr6CnD5TMAkBMi4gy7n4LLtAzM0hYWaGa4C14EJVDEBF279K91+RLgWaPgxO7FenY+EMzAwIQs1A2FyVkzAkgRw33LJht9wn6e4oyIcer6k1wuUUexYqBlFkvNhAhYlUPLTYcW2oGj1sZd2uka8/Q3R9EanY6FmNFHOjwlYiEApWoDWpWhLbZVss08uzkBDhOYILNTMcR24Fkwg6ARImJX+clRXuWjd9gga6ZqpKxODE7PpIYOBcnZMwAoEHBnZULJdvyENSZYn5azdZSpPRizUrHAHch2ZgIEESn4+Ao49J3TnSA5VCtdmLtCdkcEZsFAzGChnxwTMSkAucUDJ5oN69nX+1TQJiiLCwyYU/XXENL9YqJnmUnBFmEDgCJBSbfHG/SAXlBpUiDTx3Le7Mw3KzNBsWKgZipMzYwLmIkC9s9IdRxVXkCK+Bjy1BhcHPkVPUW95OmeGYyzUzHAVuA5MwGgC6DCo9LdccOw8Jm7s0XOdMmPjbfeacdjpri4LNTcJ/mYCoUAAdwU49udCKS4EyEVGDTXdYKRzYWG2G8ykk+auWdlvFmplafBvJmBFAmhUw0l+ag+cBsdhNBuEbhsND5JULNltw8+vythheN4GZ8hCzWCgnB0TCDiBP33ROrPR6XbOOXS8nQ9yifi+TRX1daEH0FsLV2VUqUMVFfVUoogLNbS5RPouHJgAEwggAfTqRI6zZQc6QUHBJaMzbVdBSQALrJS1jIbSxhemZ35e6YxJD4gLNXKhZYACn0m5cLWYABOQJAeudI4tWrN3rpVgiAs1K7WS68oEmIBWAgV2SRpZsGbv11oTVnV8FmpVfQW4fCZgMgK4/SlHkuRrUKBtMlnVVFWHrXSowsSRmED1IIAC7TvJLnUuTM+ypECjq8RCrXrcq9xKJuCPgIzGHmcMq9epb+HqvZZeAeThp79LzeeZQIgTwN7Zb3IYjCtalfntAsiyfGu5p2b5S8gNYAKiBKQSXN18NrFeYmoxCjTRXMyWjntqZrsiXB8mEAQC6ChlRbjd/kDeqj17D4MpjW0IU2ChJoyOEzIBCxKQpKU2u/25wlV7Npp5U7oesizU9NDjtEzAEgSkUgnkL1DvbMb59MztlqiyjkqyUNMBj5MyATMTwAWALbIkzY0A+DQvPfOkmetqZN1YqBlJk/NiAlVLoAAF2feoOLs6DGxf5KXv3UPVKa7aOgW9dBZqQUfOBTIBAwhI0nm0ZLsXJ/wzMLddqHG6rmWdiE27FuwK6m53A1pieBZhSTmRpbk26THDc+YMmQAT0E9AhmJZkvMkF+ShPbN8WZbOgew8iFuYjqA6Bprw+Cvs+utntf71/4wQBbx5YvjXAAAAAElFTkSuQmCC"></image></g></g></g></svg>
```

Comment:
Just confirming if this is a minify SVG.

Reply:
@Satendra-SR I have corrected that one extra space. For this one, I had to minify it manually as after minify from the various tools the image was getting corrupted.